atmos_model.F90

```diff
@@ -1706,6 +1717,9 @@ subroutine assign_importdata(rc)
             call ESMF_FieldGet(importFields(n),farrayPtr=datar82d,localDE=0, rc=rc)
             if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
             datar8 = datar82d
+            if (merge_import) then
+              mergeflg(:,:) = datar82d(:,:).eq.missing_value
```

Comment:
Will mergeflg need only be set up once?

Reply:
Missing values are cells outside the remapping/interpolation boundary. If configured properly then ESMF regridding will leave the indicator value of 9.99e20. If the domain ever moves then mergeflg needs to be calculated every time the domain moves (either at the source, internal to the atm, or the landsea mask changes). If variables ever use different domains then mergeflg needs to be computed for each variable. I'm open to other suggestions for merge_import and mergeflg. Also, if the application does not use CMEPS and the domain is changing then the import fields need to be reset to 9.99e20 each time the domain moves.

Reply:
@danrosen25 and @junwang-noaa, there are moving-nesting capability under development, and in the future potentially we could couple the moving nesting grid with other model component.